### ISSUE TITLE
There is no value data 

### ISSUE THREAD

**Issue by A-Pai** [barcaroli/QGA#6]
https://barcaroli.github.io/QGA/articles/QGA.html
![image](https://github.com/user-attachments/assets/40864add-f4d6-49b5-8dda-2084d01cb41a)

![image](https://github.com/user-attachments/assets/50c670b6-7cb6-4858-aa3f-4bb8163d012f)

**Comment by barcaroli** [comment]
You are right, the term "value" is misleading. It should be "identifier" (the variable "Item").

**Comment by A-Pai** [comment]
> You are right, the term "value" is misleading. It should be "identifier" (the variable "Item").

--you means the variable "Item" is value？but it is not numerical
